src/main/java/edu/hawaii/its/api/service/GroupingOwnerService.java

```diff
@@ -50,6 +51,9 @@ public GroupingGroupsMembers paginatedGrouping(String currentUser, List<String>
                 isAscending);
         GroupingGroupsMembers groupingGroupsMembers = new GroupingGroupsMembers(getMembersResults);
         groupingGroupsMembers.setPageNumber(pageNumber);
+        if(grouperService instanceof GrouperApiOOTBService && pageNumber > 1){
+            groupingGroupsMembers.setPaginationCompleteTrue();
+        }
```

Comment:
I know you encountered problems with the paginationComplete value, but could you investigate a way to avoid adding this if statement? If this if statement turns out to be the best option, then I'm ok with keeping it.

Reply:
Oh yeah, I'm implementing the paginated function for ootb that has same thing in grouperClient to get rid of that. I will remove that in later tickets.